### PR TITLE
DOCS-166: update docs nav for dev-tools

### PR DESCRIPTION
**Pull request by kay-kim**
Update Docs top nav bar to have Developer Tools instead of Client Libraries to allow for SDK
Screen Shot 2020-09-15 at 4 14 55 PM

- [x] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).

### CODE REVIEW

File: docs/api/_templates/docs-navbar.html
Comment (Vinnl) [inline docs/api/_templates/docs-navbar.html:23]: @kay-kim Given that https://github.com/inrupt/docs-client-lib-js/pull/9 says "Wait until next week", should this wait as well?

Comment (kay-kim) [reply docs/api/_templates/docs-navbar.html:23]: @Vinnl  -- Oh - yes please.

**Review by Vinnl — CHANGES_REQUESTED** [review]
Holding off until Sept 23rd to merge this.

**Review by Vinnl — APPROVED** [review]

**Pull request merged**
